src/include/storage/page/hash_table_block_page.h

```diff
@@ -24,8 +24,79 @@ namespace bustub {
 /**
  * Store indexed key and and value together within block page. Supports
  * non-unique keys.
+ *
+ * Block page format (keys are stored in order):
+ *  ----------------------------------------------------------------
+ * | KEY(1) + VALUE(1) | KEY(2) + VALUE(2) | ... | KEY(n) + VALUE(n)
```

Comment:
This is concatenation and not addition right? Maybe say that.

Reply:
I added a comment.